packages/editor/src/panels/files/toolbar.tsx

```diff
@@ -325,9 +334,10 @@ export default function FilesToolbar() {
                   NotificationService.dispatchNotify(err.message, { variant: 'error' })
                 })
             }
+            data-testid="files-panel-upload-folder-button"
```

Comment:
this is duplicate `data-testid` with the same value. unique ids are needs to ensure specific identification.